### PR TITLE
Use REDIS TLS in case it is present but ignore the certificate verification

### PR DESCRIPTION
**Pull request by abpostelnicu**

### CODE REVIEW

File: libmozevent/utils.py [hunk 1]
```diff
@@ -146,6 +146,12 @@ class AsyncRedis(object):
     async def connect():
         if AsyncRedis.redis.get(None) is None:
             AsyncRedis.redis.set(
-                await aioredis.from_url(os.environ["REDIS_URL"], decode_responses=False)
+                await aioredis.from_url(
+                    os.getenv("REDIS_TLS_URL", None)
+                    if os.getenv("REDIS_SSL_URL", None) is not None
+                    else os.getenv("REDIS_URL", None),
```
Comment (suhaibmujahid) [inline libmozevent/utils.py:152]: It could be better for readability if this got assigned to a variable first.

File: libmozevent/utils.py [hunk 2]
```diff
@@ -146,6 +146,12 @@ class AsyncRedis(object):
...
+                    decode_responses=False,
+                    ssl_cert_reqs=None,
+                )
```
Comment (suhaibmujahid) [inline libmozevent/utils.py:155]: It seems that this caused an error:

https://community-tc.services.mozilla.com/tasks/B00xAklaTmGPfAb4pD-A1Q/runs/0/logs/public/logs/live.log#L282

File: libmozevent/utils.py (hunk 1 above)
Comment (abpostelnicu) [reply libmozevent/utils.py:152]: I will have to do it in another way because it doesn't support `ssl_cert_reqs `

File: libmozevent/utils.py [hunk 3]
```diff
@@ -146,6 +146,15 @@ class AsyncRedis(object):
     async def connect():
         if AsyncRedis.redis.get(None) is None:
             AsyncRedis.redis.set(
-                await aioredis.from_url(os.environ["REDIS_URL"], decode_responses=False)
+                await aioredis.from_url(
+                    os.getenv("REDIS_TLS_URL", None),
```
Comment (marco-c) [inline libmozevent/utils.py:150]: Nit: you already checked it's not None, so you can just do `os.environ["REDIS_TLS_URL"]`

File: libmozevent/utils.py [hunk 4]
```diff
@@ -146,6 +146,15 @@ class AsyncRedis(object):
...
+                    decode_responses=False,
+                    ssl_cert_reqs=None,
+                )
+                if os.getenv("REDIS_TLS_URL", None) is not None
+                else await aioredis.from_url(
+                    os.getenv("REDIS_URL", None),
```
Comment (marco-c) [inline libmozevent/utils.py:156]: Nit: if we're here REDIS_URL must be there, so you can just do `os.environ["REDIS_URL"]`

**Review by marco-c — APPROVED** [review]

**Pull request merged**